packages/api/graphql/src/query/members/list.mock.ts

```diff
@@ -8,6 +8,86 @@ export const data = [
+      },
+      {
+        name: 'Space name',
+        logo: 'https://s3-alpha-sig.figma.com/img/43bc/b5bb/3fde1f75e375e7852efe7651b1ffe90a?Expires=1737936000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=iPI3x2cKxy8buIlOT1uaWD9gCJky7mR-9PeouWW8KJmFWE-Zd2Axp2DjaEgNm6f3cuyVA4UW9rYxd~uWEMM8tLE1doiDCZVnFTI1lYBFMAp92Jhn5YXFOIqb5MQ7sfhWGEOOxPdkF2Q6ZPoS5KRlfCC9MQuArQVbHBMVYAouDyyzQD9Y~LLIoCR1nI0~i2ur8WvbnX968zJI4SHgEsjKE11OWzNRHL4axkRWW3V6N9Yslb3TANDUkj6A2fE-9uZnzaeHmji99v8RAog1GUOkbzICQEL2Xclok9TyM~fI0jftyq3glLtRrJsCrNQbXnMIJ9fPtytavHTA944s24YT7A__',
+      },
+      {
+        name: 'Space name',
+        logo: 'https://s3-alpha-sig.figma.com/img/43bc/b5bb/3fde1f75e375e7852efe7651b1ffe90a?Expires=1737936000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=iPI3x2cKxy8buIlOT1uaWD9gCJky7mR-9PeouWW8KJmFWE-Zd2Axp2DjaEgNm6f3cuyVA4UW9rYxd~uWEMM8tLE1doiDCZVnFTI1lYBFMAp92Jhn5YXFOIqb5MQ7sfhWGEOOxPdkF2Q6ZPoS5KRlfCC9MQuArQVbHBMVYAouDyyzQD9Y~LLIoCR1nI0~i2ur8WvbnX968zJI4SHgEsjKE11OWzNRHL4axkRWW3V6N9Yslb3TANDUkj6A2fE-9uZnzaeHmji99v8RAog1GUOkbzICQEL2Xclok9TyM~fI0jftyq3glLtRrJsCrNQbXnMIJ9fPtytavHTA944s24YT7A__',
+      },
+      {
+        name: 'Space name',
+        logo: 'https://s3-alpha-sig.figma.com/img/43bc/b5bb/3fde1f75e375e7852efe7651b1ffe90a?Expires=1737936000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=iPI3x2cKxy8buIlOT1uaWD9gCJky7mR-9PeouWW8KJmFWE-Zd2Axp2DjaEgNm6f3cuyVA4UW9rYxd~uWEMM8tLE1doiDCZVnFTI1lYBFMAp92Jhn5YXFOIqb5MQ7sfhWGEOOxPdkF2Q6ZPoS5KRlfCC9MQuArQVbHBMVYAouDyyzQD9Y~LLIoCR1nI0~i2ur8WvbnX968zJI4SHgEsjKE11OWzNRHL4axkRWW3V6N9Yslb3TANDUkj6A2fE-9uZnzaeHmji99v8RAog1GUOkbzICQEL2Xclok9TyM~fI0jftyq3glLtRrJsCrNQbXnMIJ9fPtytavHTA944s24YT7A__',
+      },
+      {
+        name: 'Space name',
+        logo: 'https://s3-alpha-sig.figma.com/img/43bc/b5bb/3fde1f75e375e7852efe7651b1ffe90a?Expires=1737936000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=iPI3x2cKxy8buIlOT1uaWD9gCJky7mR-9PeouWW8KJmFWE-Zd2Axp2DjaEgNm6f3cuyVA4UW9rYxd~uWEMM8tLE1doiDCZVnFTI1lYBFMAp92Jhn5YXFOIqb5MQ7sfhWGEOOxPdkF2Q6ZPoS5KRlfCC9MQuArQVbHBMVYAouDyyzQD9Y~LLIoCR1nI0~i2ur8WvbnX968zJI4SHgEsjKE11OWzNRHL4axkRWW3V6N9Yslb3TANDUkj6A2fE-9uZnzaeHmji99v8RAog1GUOkbzICQEL2Xclok9TyM~fI0jftyq3glLtRrJsCrNQbXnMIJ9fPtytavHTA944s24YT7A__',
+      },
+      {
+        name: 'Space name',
+        logo: 'https://s3-alpha-sig.figma.com/img/43bc/b5bb/3fde1f75e375e7852efe7651b1ffe90a?Expires=1737936000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=iPI3x2cKxy8buIlOT1uaWD9gCJky7mR-9PeouWW8KJmFWE-Zd2Axp2DjaEgNm6f3cuyVA4UW9rYxd~uWEMM8tLE1doiDCZVnFTI1lYBFMAp92Jhn5YXFOIqb5MQ7sfhWGEOOxPdkF2Q6ZPoS5KRlfCC9MQuArQVbHBMVYAouDyyzQD9Y~LLIoCR1nI0~i2ur8WvbnX968zJI4SHgEsjKE11OWzNRHL4axkRWW3V6N9Yslb3TANDUkj6A2fE-9uZnzaeHmji99v8RAog1GUOkbzICQEL2Xclok9TyM~fI0jftyq3glLtRrJsCrNQbXnMIJ9fPtytavHTA944s24YT7A__',
+      },
+    ],
+    agreements: [
```

Comment:
Let's not combine those apis. I think it's easier to have seperate api endpoints for the different entities. What do you think?